config/environments/development.rb

```diff
@@ -90,7 +90,7 @@ module OptimizerWrapper
       batch_heuristic: ENV['OPTIM_DBG_BATCH_HEURISTIC'] == 'true'
     },
     restitution: {
-      allow_polylines: ENV['OPTIM_GENERATE_GEOJSON_POLYLINES'] == 'true'
+      allow_polylines: ENV['OPTIM_GENERATE_GEOJSON_POLYLINES'] != 'false'
```

Comment:
Why changing this condition and only this one ?

Reply:
The others are debug outputs that are activated for debugging purposes but this is related to the vrp/json file itself. On local we never launch `polylines:true` or `polylines:[...]` unless we really need the geojson; however, if you forget to launch the server with `OPTIM_GENERATE_GEOJSON_POLYLINES` set then you get nothing -- this creates unnecessary friction.  This env var was originally created as a protection for the servers so I reverted on only local.